test/test_coupling.jl

```diff
@@ -95,17 +97,17 @@ for L = 0:Lmax
                        == rank(gram(Ure), rtol = 1e-12) 
                        == rank(gram(Ure_r), rtol = 1e-12) ) )
 
-      X = [ rand_sphere() for i in 1:length(ll) ]
+      X = [ rand_ball() for i in 1:length(ll) ]
```

Comment:
Why change to rand_ball? When the vector is near zero, then the ≈ tests could potentially fail because they enforce relative errors. It depends on the basis of course but this is a potential source for testing bugs. 

I would prefer reverting to sphere, but if there is a reason ball is needed let me know.

Reply:
I just think that by using `rand_sphere`, the tests were not entirely complete - we didn't really test the radial part. But I can add something so that the atoms are away from the origin. I think it has been done elsewhere before, so perhaps just a copy-paste.

Reply:
I hacked a bit from `rand_x` in `simple_ace.jl`